components/graphs/StatusDonut.js

```diff
@@ -58,13 +54,13 @@ const StatusDonut = (props) => {
     values: [...data.values],
     labels: [...data.labels],
     domain: { column: 0 },
-    hoverinfo: 'label+value+percent',
-    hovertemplate: '%{label}<br>%{value}<br>%{percent:.0%}<extra></extra>',
+    hoverinfo: "label+value+percent",
```

Comment:
We (loosely) follow [Airbnb jsx styleguide](https://airbnb.io/javascript/react/#quotes) hence single quotes preferred here.

Maybe look into adding [Eslint](https://eslint.org/) in your code editor? The eslint settings with the preferred code style are included in our package.

Reply:
Changed the quotes to single